### DISCUSSION TITLE
Phosani's Nightmare icon on Hiscores

### DISCUSSION THREAD

**Discussion by Lars4street** [runelite/runelite#15693] [Ideas]
Both Nightmares are the only ones with the same icon seen through the Lookup option from the Hiscore plugin, so I suggest changing the Phosani's one for the Parasitic Egg icon, as seen on the official Hiscore page, for consistency.

![Screenshot_904](https://user-images.githubusercontent.com/29815772/190887115-6fd36c80-5f29-4196-b3dd-9d3a37e8dea3.png)

![Screenshot_903](https://user-images.githubusercontent.com/29815772/190886978-4a0437b4-13bd-401e-ae85-895a02c31d51.png)

**Comment by Lars4street** [comment]
bump

**Comment by Felanbird** [comment]
Fixed in 0304007f4e3c61eab0d7c1b1e6947c8957426bda
